FLIMfitFrontEnd/Classes/@flim_data_series/compute_tr_data.m

```diff
@@ -156,35 +148,9 @@
             in = in(1,1,:,:) + in(1,2,:,:); % 2*obj.g_factor*
         end
      
-        size_before = size(in);
-        squeezed = squeeze(in);
-        
-        if length(size_before) == 4 && size_before(3) == 1
-            % handle possible  dimension flip when squeezing 1xn data
-            if size_before(4) && size(squeezed,1)
-                squeezed = squeezed';
-            end
-        end
-        
-        obj.intensity = squeezed;
- 
-        %sz = size(obj.cur_tr_data);
-        
-        %in = reshape(obj.cur_tr_data,[sz(1) prod(sz(2:end))]);
-        
-        %s = sum(in,2);
-        
-        %sel = s > 0.5 * max(s);
-        
-        %{
-        in = obj.cur_tr_data(sel,1,:,:);
-        in = sum(in,1);
-        in = sum(in,2);
-        in = squeeze(in);
-        
-        %obj.intensity = in;
-        %}
+        obj.intensity = reshape(in,obj.data_size(3:4)');

```

Comment:
It's computing the intensity from FLIM data - it's already been summed over the first two dimensions so what's left is a 1x1xnxm image and we want to get a nxm image